### PR TITLE
adaptions to testing code to match ros-planning/moveit_resources#7

### PR DESCRIPTION
**Pull request by rhaschke**
Depending on https://github.com/ros-planning/moveit_resources/pull/7 this is the second part of https://github.com/ros-planning/moveit/issues/17

- adapting source to match the renaming of test -> pr2_descriptionn in https://github.com/ros-planning/moveit_resources/pull/7

UPDATE: The integration tests will be filed as a separate PR. They should be disabled for now anyway, because they are blocked by:
    - [`pyassimp` failure in `kinetic`](https://travis-ci.org/ros-planning/moveit/builds/153589357): **That's a `moveit_commander` regression!**
    - cleanup.test: https://github.com/ros/ros_comm/pull/871
    - validate_traj.test: https://github.com/ros-planning/moveit/pull/63
    - robot_state_update.test: https://github.com/rhaschke/moveit/pull/1

### CODE REVIEW

File: moveit_ros/integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(moveit_ros_integration_tests)
```
Comment (davetcoleman) [inline moveit_ros/integration_tests/CMakeLists.txt]: I think we should move away from the moveit_ros subfolders eventually. Why not just call this ``moveit_integration_tests`` and leave in root folder?

Comment (rhaschke) [inline moveit_ros/integration_tests/CMakeLists.txt]: These are actually ROS-related integration tests. That's why I decided to put them into moveit_ros originally.

Comment (davetcoleman) [inline moveit_ros/integration_tests/CMakeLists.txt]: This isn't a big deal, but don't you think its redundant - what other type of integration test could we have? We could have non-ROS unit tests, but ROS is what MoveIt! uses for all high level integration

Comment (rhaschke) [inline moveit_ros/integration_tests/CMakeLists.txt]: That true, of course. However, the name underlines, that those integrations tests require a ROS environment ;-)

**Comment by rhaschke** [comment]
There is an open issue: Some moveit_core tests fail due to changes in https://github.com/ros-planning/moveit_resources/pull/7. Will have a look.

**Comment by davetcoleman** [comment]
future reference: would be nice if there were separate PR2 for what this includes: 1) moving the old pr2 tests because of moveit_resources change, and 2) adding new tests

**Comment by davetcoleman** [comment]
in fact - i think you should separate this PR because part 1) must be cherry-picked for I/J too since moveit_resources only has a master branch. then you could work on all the new features as step 2

**Comment by rhaschke** [comment]
@davetcoleman I have split the PR as requested, keeping part 1 here and opening a new one for part 2.
I cancelled the [Travis check](https://travis-ci.org/ros-planning/moveit/builds/155157619) on purpose, because it will fail due to https://github.com/ros-planning/moveit_resources/pull/7 not yet merged.
Instead, I have triggered a Travis build here https://travis-ci.org/rhaschke/moveit/builds/155158290, which uses the new version of `moveit_resources`.

**Comment by rhaschke** [comment]
Dave, if you happen to merge this PR, please change the commit comment to not _mention integration tests_, which is pulled in by the branch name if I remember correctly.
I cannot change the name of the branch without closing the PR.

File: moveit_core/collision_detection_fcl/test/test_fcl_collision_detection.cpp [hunk 2]
```diff
@@ -61,16 +61,10 @@ class FclCollisionDetectionTester : public testing::Test
...
-    if(resource_dir == "")
-    {
-      FAIL() << "Failed to find package moveit_resources.";
-      return;
-    }
-    boost::filesystem::path res_path(resource_dir);
-    std::string urdf_file = (res_path / "test/urdf/robot.xml").string();
-    std::string srdf_file = (res_path / "test/srdf/robot.xml").string();
-    kinect_dae_resource_ = "package://moveit_resources/test/urdf/meshes/sensors/kinect_v0/kinect.dae";
+    boost::filesystem::path res_path(MOVEIT_TEST_RESOURCES_DIR);
```
Comment (davetcoleman) [inline moveit_core/collision_detection_fcl/test/test_fcl_collision_detection.cpp]: this is slick!

**Comment by davetcoleman** [comment]
Thanks for splitting this PR up, much easier to review that way.

**Comment by davetcoleman** [comment]
Once https://travis-ci.org/ros-planning/moveit/builds/155413640 passes to verify kinetic branch is stable, I'll cherry pick to I/J

**Comment by davetcoleman** [comment]
Passed, cherry-picked to I/J

**Pull request merged**